CHARTER.md

```diff
@@ -0,0 +1,107 @@
+# Technical Charter for Open Source Security Foundation
+
+[Securing Critical Projects - Working Group]
+
+In Review August 18th, 2022
+
+This Technical Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the Securing Critical Projects WG open source community, which has been established as a Working Group (the "Technical Initiative") under the Open Source Security Foundation (the “OpenSSF”).  All contributors (including committers, maintainers, and other technical positions) and other participants in the Technical Initiative (collectively, “Collaborators”) must comply with the terms of this Technical Charter and the OpenSSF Charter. 
+
+#### 1. Mission and Scope of the Technical Initiative
+
+- a. The mission of the Technical Initiative is to identify critical open source software (OSS) projects and secure those projects. This space allows members to collaborate together on these goals.
+
+- b. The scope of the Technical Initiative includes collaborative development under the Technical Initiative License (as defined herein) supporting the mission, including organizing collaboration activities, defining best practices, documentation, testing, integration, and the creation of other artifacts that support the mission.
+
+#### 2. Technical Steering Committee
+
+- a. The Technical Steering Committee (the "TSC") will be responsible for all oversight of the Technical Initiative. 
+
+- b. The TSC voting members are initially the Technical Initiative’s Maintainers. The Maintainers will be documented in the Technical Initiative repository. The TSC is responsible for determining the future process for defining voting members of the TSC, and any such alternative approach will also be documented appropriately.  Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person. 
```

Comment:
I would also add in here that while they intend all meetings to be open to the public, the TSC reserves the right to make a given meeting closed to handle sensitive issues that arise.